### DISCUSSION TITLE
Is Vuefire broken again in Nuxt 3.8.0, I remember this issue was fixed for Nuxt 3.7.0

### DISCUSSION THREAD

**Discussion by fayazara** [vuejs/vuefire#1454] [Questions and Help]
Using  
`"vuefire": "^3.1.17"`
`"nuxt-vuefire": "^0.4.0",`
`"nuxt": "^3.8.0",`

I saw a similar issue (https://github.com/vuejs/vuefire/issues/1421) for nuxt 3.7.0 which @posva had fixed.

But I am still getting the same error 

The default Firebase app does not exist. Make sure you call initializeApp() before using any of the Firebase services.
![Screenshot 01-11 at 17](https://github.com/vuejs/vuefire/assets/15716057/a2bc26ac-b87e-4e5c-b8d9-038f09cc6663)

**Comment by dosstx** [comment]
Do u have a cloud functions folder?
